misc/handlers.go

```diff
@@ -140,17 +141,48 @@ func handlerAbout(w http.ResponseWriter, rq *http.Request) {
 var stylesheets = []string{"default.css", "custom.css"}
 
 func handlerStyle(w http.ResponseWriter, rq *http.Request) {
-	w.Header().Set("Content-Type", mime.TypeByExtension(".css"))
+	var filesData []io.Reader
+
+	var latestModTime time.Time
+
+	// Step 1: Collect files data and determine the latest modification time
 	for _, name := range stylesheets {
 		file, err := static.FS.Open(name)
 		if err != nil {
 			continue
 		}
-		_, err = io.Copy(w, file)
+
+		fileStats, err := file.Stat()
 		if err != nil {
+			continue
+		}
+
+		modTime := fileStats.ModTime()
+		if modTime.After(latestModTime) {
+			latestModTime = modTime
+		}
+
+		filesData = append(filesData, file)
+
+		defer file.Close()
```

Comment:
defer in for cycle it’s not good idea. 

File will be close only on exit from func